core/lib/basic_types/src/protocol_version.rs

```diff
@@ -255,3 +278,106 @@ impl From<ProtocolVersionId> for VmVersion {
         }
     }
 }
+
+basic_type!(
+    /// Patch part of semantic protocol version.
+    VkPatch,
```

Comment:
Also, why tie specifically to `VkPatch`? I think that a "patch" is generally "a version change that should not affect user-facing behavior". One of the examples of how we could do a patch without changing vks is if we had to somehow amend the priority queue in an emergency

Reply:
On a server side patch implies only vk changes, so I came up with this name. Do you think it makes sense to bump patch if the upgrade doesn't change server/prover params? I don't see reasons for that, but if you think so I can amend strcut name and eth sender behavior a bit which currently assumes that each patch changes verification keys

Reply:
If not hard, I would leave our options open just in case. I agree that 99% of the time those will be VKs, but one of the reasons we started the whole patch thing is so that we could always reuse the same `DefaultUpgrade` contract that requires that the protocol version number goes up